### PR TITLE
corrections in reconnecting automation

### PR DESCRIPTION
**Pull request by elschnert**
## Proposed change
Fixed issues in the example reconnecting automation and the corresponding template

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @elschnert, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20884).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/homematic.markdown [hunk 1]
```diff
@@ -485,16 +485,16 @@ template:
   - binary_sensor:
       - name: "Homematic is sending updates"
         state: >-
-          {{ now() - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
+          {{ as_timestamp(now()) - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
```
Comment (frenck) [inline source/_integrations/homematic.markdown]: We don't need to calculate using `as_timestamp` anymore, just use `as_datetime`, which can be subtracted from `now()` directly, resulting in a diff.

Additionally, I don't think you can get `last_changed` using `state_attr` actually

Comment (elschnert) [reply source/_integrations/homematic.markdown]: Good to know, i will update the code to use as_datetime.

Indeed I was not able to get the `last_changed` using `state_attr` and ended up using `states.sensor['xyz].last_changed` which felt like a workaround - so i left it unchanged. Will update to my solution.

File: source/_integrations/homematic.markdown [hunk 2]
```diff
@@ -485,16 +485,16 @@ template:
   - binary_sensor:
       - name: "Homematic is sending updates"
         state: >-
-          {{ now() - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
+          {{ (now() - states.sensor['office_voltage'].last_changed).seconds < 600 }}
```
Comment (frenck) [inline source/_integrations/homematic.markdown]:
```suggestion
          {{ (now() - states.sensor.office_voltage.last_changed).seconds < 600 }}
```

Comment (elschnert) [reply source/_integrations/homematic.markdown]: This doesn't work in my case, as the given sensor has a generated entitiy-id is starting with numbers. But for the documentation this is definetly the cleaner version.

Comment (frenck) [reply source/_integrations/homematic.markdown]: Uh, so that should work?

**Review by frenck — APPROVED** [review]
Thanks, @elschnert 👍

File: source/_integrations/homematic.markdown (hunk 2 above)
Comment (elschnert) [reply source/_integrations/homematic.markdown]: I'm using an homematic ccu3. The generated id's look like `00095be98bXXXX_illumination`.

If I use this in a template
`{{ (now() - states.sensor.00095be98bXXXX_illumination.last_updated).seconds < 3600 }}`

I run into
`Invalid config for [template]: invalid template (TemplateSyntaxError: expected token ')', got 'integer') for dictionary value @ data['binary_sensor'][0]['state']. Got '{{ (now() - states.sensor.00095be98bXXXX_illumination.last_updated).seconds < 3600 }}'`

**Pull request merged**